AaSystem/LogAndPrint/Log.py

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/python3
```

Comment:
prefer specific imports than *
i.e. `from AaSystem.LogAndPrint import Colors`

Reply:
I dont think it makes much of a difference in this case since there is only one thing inside the entire file which is colors but sure, updated